src/Javis.jl

```diff
@@ -1407,6 +1420,7 @@ export javis, latex
 export Video, Action, BackgroundAction, SubAction, Rel
 export Line, Translation, Rotation, Transformation, Scaling
 export val, pos, ang, get_value, get_position, get_angle
+export get_javis_frame
```

Comment:
I don't think we want to export this but if we do it needs better documentation like an example and explaining the arguments.

Reply:
I actually do want to have this export as I think it would be useful to have. Plus, it has been somewhat requested on Zulip and in issues #91 and #92 . I don't know what a good example would be for this. Any ideas?

Reply:
I think it actually can't be called when `javis` isn't called before. We should therefore have a preprocess  function which then can be exported as well. But not in this PR. For now I would not export it and the users can do Javis.get_javis_frame it show that this might change  in the future.